.github/workflows/welcome.yml

```diff
@@ -0,0 +1,41 @@
+# Greet first-time contributors.
+
+name: Greet First-Time Contributors
+
+on:
+  pull_request:
+    types:
+      - opened
+  issues:
+    types:
+      - opened
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  greeting:
+    name: Greet First-Time Contributors
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/first-interaction@v3.1.0
+        with:
+          issue_message: |
+            👋 Thanks for opening your first issue here! Please make sure you filled out the template with as much detail as possible. We appreciate that you took the time to contribute!
+
+            Please make sure you read our [Contributing Guide](https://github.com/GenericMappingTools/gmt/blob/master/CONTRIBUTING.md) and abide by our [Code of Conduct](https://github.com/GenericMappingTools/.github/blob/main/CODE_OF_CONDUCT.md).
+
+          pr_message: |
+            💖 Thanks for opening this pull request! 💖
+
+            Please make sure you read our [Contributing Guide](https://github.com/GenericMappingTools/gmt/blob/master/CONTRIBUTING.md) and abide by our [Code of Conduct](https://github.com/GenericMappingTools/.github/blob/main/CODE_OF_CONDUCT.md).
+
+            A few things to keep in mind:
+
+            * If you need help writing tests, take a look at the existing ones for inspiration. If you don't know where to start, let us know and we'll walk you through it.
+            * All new features should be documented. It helps to write the comments below for your functions describing the it, **all** arguments, and return types before writing the code. This will help you think about your code design and results in better code.
```

Comment:
Replace
* All new features should be documented. It helps to write the comments below for your functions describing the it, **all** arguments, and return types before writing the code.
by
* All new features should be documented. It helps to write the comments below for your functions describing the it, **all** arguments.

The return type in C are already mandatory in function definition.